pcl_ros/tools/pcd_to_pointcloud.cpp

```diff
@@ -89,46 +94,48 @@ class PCDGenerator
       if (file_name_ == "" || pcl::io::loadPCDFile (file_name_, cloud_) == -1)
         return (-1);
       cloud_.header.frame_id = tf_frame_;
+      // set timer to publish cloud periodically
+      timer_ = nh_.createWallTimer(
```

Comment:
preference to timer would be to convert publisher to latched, and make service call trigger publishing the new point cloud